components/org.wso2.carbon.config/src/main/java/org/wso2/carbon/config/ConfigurationUtils.java

```diff
@@ -159,4 +159,27 @@ public static String getSystemVariableValue(String variableName, String defaultV
         }
         return value;
     }
+
+    /**
+     * Returns replace value with escaped characters.
+     * @param value replaced values
+     * @return value with escaped characters
+     */
+    public static String escapeSpecialCharacters(String value) {
+        if (value == null) {
+            return null;
```

Comment:
This can cause a `NullPointerException`. Shouldn't we return a blank string ("") instead?

Reply:
Intention is to pass the same value if the value is null. We don't need to do anything in this utils method. if the value is null. Getting null value  means there is a problem in the system.